src/common/Utility.js

```diff
@@ -237,6 +237,53 @@
                     return node;
                 }
             };
+        },
+        downloadData: function (format, data, id) {
+            id = id || "data";
+            var currentdate = new Date(); 
+            var timeFormat =  d3.time.format("%m-%d-%Y_%H.%M.%S");
```

Comment:
Can you switch this to:  ```%Y-%m-%dT%H:%M:%S``` (sorry for being picky) - but:
1.  Its sortable
2.  It matches time stamps we use elsewhere
3.  The "." in the time could mess with the file extension.